fetchai/ledger/api/contracts.py

```diff
@@ -68,24 +57,19 @@ def query(self, contract_owner: Address, query: str, **kwargs):
         return self._post_json(query, prefix=str(contract_owner), data=self._encode_json_payload(**kwargs))
 
     def action(self, contract_address: Address, action: str,
-               fee: int, from_address: Address, signers: EntityList,
-               *args, shard_mask: BitVector = None):
```

Comment:
The difference here seems to be that we have swapped the order in the params list of signers and args, is there any point to that?